### ISSUE TITLE
Ghost-paper disrupts scroll behavior of page

### ISSUE THREAD

**Issue by constantin-huetterer** [amcharts/amcharts4#3363]
Our company purchased a License for amCharts a while ago and I really like using it. Great job on the library! =) 

However, I encountered an issue today, where our page layout was "expanded" beyond the boundary of the root-HTML element. To illustrate the issue, I put in `html {border: 3px solid red;}`. This is the outcome:
![image](https://user-images.githubusercontent.com/60255589/110359262-92409b00-803d-11eb-9442-48f8b2a8b64a.png)
 
The invisible ghost paper element sits at the bottom of the page and therefore the scrollbar indicates that there is more content on the page than there actually is. After some debugging, I found out that amcharts seems to be the culprit of this problem. In particular the "ghost paper" that is added to the body element of the document. 

https://github.com/amcharts/amcharts4/blob/1b18099adea5b224b3583b9b5ab8032c05a89423/src/.internal/core/rendering/Paper.ts#L185

For me it is unclear why the element is added "outside" of the html root, causing this overflow issue. When I set html to relative and give the ghost paper the top position 0, the issue is resolved, since it is now moved to the top instead of the bottom of the page. However this doesn't really resolve the issue. 

Please let me know if I can assist further with fixing this issue.

**Comment by martynasma** [comment]
It is added to the document root (not to the chart's own div) because ghost paper is shared across multiple charts, so it's just one on the page.

As for the solution, I think I have an idea: shifting the ghost paper up by one pixel or by setting top/left.

Would you mind trying it out if that fixes the issue with your scrollbars (I can't seem to be able to reproduce it).

You can find the ghost paper div in browser's DOM explorer and add this CSS:

```CSS
margin-top: -1px;
```

Or this:

```CSS
top: 0;
left: 0;
```

**Comment by constantin-huetterer** [comment]
Thanks for the quick response. Having looked at this with fresh eyes today, I found that the issue is in:
https://github.com/amcharts/amcharts4/blob/1b18099adea5b224b3583b9b5ab8032c05a89423/src/.internal/core/utils/Instance.ts#L43

rather than originially stated in Paper.ts. This element is similarly appended to the body of the page (but a lot larger than the 1px ghost-paper element) causing the scrolling-issue. It seems to be added multiple times as well.
![image](https://user-images.githubusercontent.com/60255589/110446700-bf359200-80bf-11eb-8d20-03c57dde9bf0.png)

Using this code fixes the problem for us:
```js
function createChild(htmlElement, classType) {
    var htmlContainer = $dom.getElement(htmlElement);
    // If there's no container available yet, we create a fake one
    var tmpContainer = false;
    if (!htmlContainer) {
        htmlContainer = document.createElement("div");
        htmlContainer.style.width = "200px";
        htmlContainer.style.height = "200px";
        htmlContainer.style.top = "0";
        htmlContainer.style.left = "0";
        htmlContainer.style.visibility = "hidden";
        htmlContainer.style.position = "absolute";
        document.body.appendChild(htmlContainer);
        tmpContainer = true;
    }

    ...
```

**Comment by martynasma** [comment]
Cool, thanks for checking.

So, essentially setting top/left, right?

I think it would be safe to add that into engine.

**Comment by constantin-huetterer** [comment]
No problem. Correct, top/left does the trick. No other changes in there. :slightly_smiling_face:

**Comment by martynasma** [comment]
Gotcha. Thanks.

We'll test this some more and will include into next version.

Will ping you here once it's out. This or next week.

**Comment by constantin-huetterer** [comment]
Great, thanks a lot. :+1:

**Comment by martynasma** [comment]
Fixed in [4.10.17](https://github.com/amcharts/amcharts4/releases/tag/4.10.17).

## [4.10.17] - 2021-03-19

### Added
- New setting for `Export`: `webFontFilter`. Allows it to set to a regular expression. If set only those external fonts that their URL match regular exporession will be loaded.
- New setting for `ForceDirectedSeries`: `showOnTick` (default `10`). Render series hidden until Xth tick.
- New setting for `XYCursor`: `snapOnPan` (default `true`). Indicates whether to "snap" zoom selection to equal periods when panning by cursor ends.

### Fixed
- Zooming of chart with multiple vertical axes of which one was inverted was incorrect.
- `minZoomCount` was not being taken into account when zooming with mouse wheel.
- JSON config: `exporting.dataFields` setting was being ignored.
- Creating chart before its container is ready, was resulting in extra element in DOM in some cases.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
